CHANGELOG.rst

```diff
@@ -2,10 +2,20 @@
 ======
 NOT RELEASED
 
+Features
+--------
+* Add Cluster ssl_context option to enable SSL (PYTHON-995)
+* Allow encrypted private keys for 2-way SSL cluster connections (PYTHON-995)
+
```

Comment:
RFC: I wrote 2 entries just for the discussion purpose. The first one is the one that makes it 100% clear to users what new feature/improvement we did in PYTHON-995. The second is simply the ticket title as we used to add in the changelog. I would like to know if you have any idea/preference how we should handle this in the future. I know we are not going to change all the jira ticket titles... but at the same time, I think it's very valuable to users to know on a quick changelog read what new features we added.

Reply:
I think we should do what the java team do: creating an additional ticket for the purpose when appropriate: 
![screenshot_20190123_153830](https://user-images.githubusercontent.com/532356/51635509-f028cc00-1f24-11e9-9cb1-25f626f7e0c1.png)

Reply:
Good. I'm going to follow this in the future then.